packages/react-native-client/src/hooks/useScreenShare.ts

```diff
@@ -94,10 +94,9 @@ export function useScreenShare() {
   const toggleScreenShare = useCallback(
     async (screenShareOptions: Partial<ScreenShareOptions> = {}) => {
       if (Platform.OS === 'android' && !isScreenShareOn) {
-        (await handleScreenSharePermission()) == 'granted' &&
-          (await RNFishjamClientModule.startForegroundService({
-            enableScreenSharing: true,
-          }));
+        if ((await handleScreenSharePermission()) != 'granted') {
```

Comment:
WDYT about enabling eslint rule that forces `===` and `!==` 🤔

Reply:
Good point! It's easy to miss this when switching between native/js side.